numpy/core/tests/test_einsum.py

```diff
@@ -274,6 +278,13 @@ def check_einsum_sums(self, dtype, do_opt=False):
             assert_equal(np.einsum(a, [0, 0], optimize=do_opt),
                          np.trace(a).astype(dtype))
 
+            #ticket #15961: should accept numpy int64 type in subscript list
```

Comment:
```suggestion
            # gh-15961: should accept numpy int64 type in subscript list
```
"Ticket" refers to a previous issue tracker